pkg/auditd/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# FROM linuxkit/alpine:630ee558e4869672fae230c78364e367b8ea67a9 AS mirror
```

Comment:
I have no idea what the promotion process is, unfortunately. I suppose we could talk to @ncopa about it :)